### PR TITLE
Us22 admin dashboard incomplete invoices

### PR DESCRIPTION
**Pull request by abwmn**

### CODE REVIEW

File: app/models/invoice.rb [hunk 1]
```diff
@@ -5,5 +5,11 @@ class Invoice < ApplicationRecord
   has_many :transactions
 
   enum status: {'in progress': 0, 'completed': 1, 'cancelled': 2}
+
+  def self.incomplete_invoices
```
Comment (thawley2) [inline app/models/invoice.rb:9]: Slick method!!

File: app/views/admin/show.html.erb [hunk 2]
```diff
@@ -13,4 +13,15 @@
       </li>
     <% end %>
   </ul>
+</section>
```
Comment (thawley2) [inline app/views/admin/show.html.erb:16]: Nice use of section, I'll have to look into trying to use more of these.

**Review by amspears007 — APPROVED** [review]
So awesome with the final match array method and rereading the story over and over:)

**Review by thawley2 — APPROVED** [review]
Nice job you two!!

**Pull request merged**